src/coreclr/jit/emitarm64.cpp

```diff
@@ -11334,6 +11382,125 @@ void emitter::emitIns_R_R_R(instruction     ins,
             fmt = IF_SVE_FO_3A;
             break;
 
+        case INS_sve_rax1:
+        case INS_sve_sm4ekey:
+            assert(isVectorRegister(reg1)); // ddddd
+            assert(isVectorRegister(reg2)); // nnnnn
+            assert(isVectorRegister(reg3)); // mmmmm
+
+            if (ins == INS_sve_rax1)
+            {
+                assert(opt == INS_OPTS_SCALABLE_D);
+            }
+            else
+            {
+                assert(opt == INS_OPTS_SCALABLE_S);
+            }
+
+            fmt = IF_SVE_GJ_3A;
+            break;
+
+        case INS_sve_fmlalb:
+        case INS_sve_fmlalt:
+            assert(isVectorRegister(reg1)); // ddddd
+            assert(isVectorRegister(reg2)); // nnnnn
+            assert(isVectorRegister(reg3)); // mmmmm
+
+            if (opt == INS_OPTS_SCALABLE_B)
+            {
+                unreached(); // TODO-SVE: Not yet supported.
+                fmt = IF_SVE_GN_3A;
```

Comment:
Interesting that this format is not able to be decoded, but `HB_3A` can. I guess it's possible.

Reply:
Yeah, according to [the docs](https://docsmirror.github.io/A64/2023-09/fmlalb_z_z8z8z8.html), that format for `fmlalb/fmlalt` was introduced with SVE2, so I'm guessing it's too new for capstone.

Reply:
Specifically it's `FEAT_FP8FMA`, which is a small feature which I don't think is in any hardware yet.